tests/publiccloud/prepare_tools.pm

```diff
@@ -27,15 +27,15 @@ sub run {
 
     if (is_sle) {
         my $modver = get_required_var('VERSION') =~ s/-SP\d+//gr;
-        add_suseconnect_product('sle-module-public-cloud', $modver);
+        add_suseconnect_product('sle-module-public-cloud', $modver, get_required_var('ARCH'));
     }
 
-
-    my $tools_repo = get_var('PUBLIC_CLOUD_TOOLS_REPO', '');
-    if ($tools_repo eq '') {
```

Comment:
hm , missed this in previous reviews :( 

don't see the point of using `get_var` with empty default value to set it just after this. Instead this 3 lines it should be just : 
```
my $tools_repo = get_var('PUBLIC_CLOUD_TOOLS_REPO', 'http://download.opensuse.org/repositories/Cloud:/Tools/' . generate_version() . '/Cloud:Tools.repo');
```

Reply:
Yes, I thought that too :) its gone anyway.